code/__DEFINES/is_helpers.dm

```diff
@@ -2,6 +2,8 @@
 
 #define isdatum(D) (istype(D, /datum))
 
+#define isWEAKREF(A) istype(A, /datum/weakref)
```

Comment:
You define this twice. Also follow the naming conventions of the other istype helpers

Reply:
Say it to Kevinz!

Reply:
fixed